app/src/main/java/com/example/moodtrackerproject/ui/login/LoginFragment.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.moodtrackerproject.ui.login
+
+import android.os.Bundle
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.core.view.isVisible
+import androidx.fragment.app.Fragment
+import com.example.moodtrackerproject.R
+import com.example.moodtrackerproject.databinding.FragmentLoginScreenBinding
+import com.example.moodtrackerproject.ui.NotesFragment
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.firebase.auth.FirebaseAuth
+
+class LoginFragment : Fragment() {
+
+    private var _binding: FragmentLoginScreenBinding? = null
+    private val mBinding get() = _binding!!
+    lateinit var auth: FirebaseAuth
+    lateinit var navBar: BottomNavigationView
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentLoginScreenBinding.inflate(layoutInflater, container, false)
+        navBar = requireActivity()!!.findViewById(R.id.bottom_navigation)
+        navBar.isVisible = false
```

Comment:
I told you you must manage bottom nav bar visibility in MainActivity depending on destination changes. Use appropriate listener method for that. You shouldn't handle it in each fragment